### DISCUSSION TITLE
Persist local file system when using `PostgreSQL` ?

### DISCUSSION THREAD

**Discussion by gcool-info** [ACINQ/eclair#2689] [Developers]
Hi there!

From [PostgreSQL.md](https://github.com/ACINQ/eclair/blob/master/docs/PostgreSQL.md):

> By default, Eclair stores its data on the machine's local file system (typically in the ~/.eclair directory) using SQLite.
>
> It also supports PostgreSQL version 10.6 and higher as a database backend.

Does this mean that the local file system storage needs to be persisted when using `PostgreSQL`?

**Imagine the following scenario**: _`eclair` runs in a container with ephemeral file system storage + `PostgreSQL` driver. The container exits and then spins up again. Files in the `eclair` container are brand new but `PostgreSQL` has persisted state._

**Comment by t-bast** [comment]
No, if you use a `PostreSQL` instance that is outside of the container, it directly holds the data, so you only need "standard" postgres backup mechanisms.

Why do you want to run `eclair` in a container in the first place though? That is likely unnecessary and adding complexity without any benefit?
